MIGRATION.md

```diff
@@ -2,35 +2,45 @@
 
 These guides below are provided to ease the transition of existing applications using the Onfido SDK from one version to another that introduces breaking API changes.
 
+## `5.0.0` -> [Next version]
+
+### Added strings
```

Comment:
I think on Android and iOS they also put the strings that have changed. Maybe double check with @anuragajwani and @gokhanaliccii

Reply:
On Android we still put [next version]

Reply:
on iOS we inform about added, changed and removed strings